tests/components/FilterSearch.test.tsx

```diff
@@ -668,6 +668,17 @@ describe('search without section labels', () => {
     userEvent.keyboard('{enter}');
     expect(inputNode).toHaveValue('first name 1');
   });
+  
+  it('when an onDropdownInputChange prop is specified, it gets called each time after the input changes and executeFilterSearch does not', async () => {
+    const mockedOnDropdownInputChange = jest.fn();
+    const executeFilterSearch = jest
+      .spyOn(SearchHeadless.prototype, 'executeFilterSearch')
+      .mockResolvedValue(unlabeledFilterSearchResponse);
+    renderFilterSearch({ searchFields: searchFieldsProp, onDropdownInputChange: mockedOnDropdownInputChange});
+    userEvent.type(screen.getByRole('textbox'), 'a');
+    await waitFor(() => expect(mockedOnDropdownInputChange).toHaveBeenCalledTimes(1));
+    await waitFor(() => expect(executeFilterSearch).toHaveBeenCalledTimes(0));
```

Comment:
nit: this `waitFor` can probably be removed since we're already waiting for the other function to have executed before this check